### DISCUSSION TITLE
REQ: <gl_outline model exclusion list + opacity option>

### DISCUSSION THREAD

**Discussion by timbergeron** [ezQuake/ezquake-source#494] [Ideas]
I like the look of gl_outline but not for every model. The idea would be to have an option to exclude certain models. See image below, the grappling hook looks strange with an outline when shot. 

![ezquake000](https://user-images.githubusercontent.com/68674590/106366910-d11f5a80-62f3-11eb-900d-7bedb939bf28.jpg)

In addition, would is be possible to change the opacity of the outline and the size of the stroke outline.

**Comment by lorenKylens** [comment]
Ich mag den Look von gl_outline, aber nicht für jedes Modell. Ideal wäre eine Option, mit der man bestimmte Modelle davon ausschließen kann. Im Screenshot sieht man z. B., dass der Enterhaken nach dem Schuss mit https://billybets-de.de Outline etwas seltsam aussieht. Wenn man einzelne Modelle von diesem Effekt ausnehmen könnte, würde das Gesamtbild deutlich sauberer wirken und der Effekt nur dort erscheinen, wo er wirklich passt.

**Reply by dsvensson** [reply]
Skulle du kunna förtydliga det på skånska?

**Comment by EdgarWright512** [comment]
That makes sense, because a global “gl_outline” effect usually applies as a post-process or shader pass to everything in the scene, so it can’t distinguish between “important” models and small gameplay objects like a grappling hook. The cleanest solution is usually adding a per-object toggle or a material flag so certain meshes are [reef spins casino](https://reef-spins3.com/) excluded from the outline pass. If you don’t control the shader, a workaround is to assign those items to a separate render layer or tag and then exclude that layer from the outline camera/pass. This keeps the visual style intact while avoiding awkward artifacts on thin or fast-moving objects.

**Comment by timbergeron** [comment]
https://github.com/timbergeron/QSS-M/blob/357e24a2aada01ebbfcafd01b25de7c2aab5d82e/Quake/r_alias.c#L988

**Comment by EverettPrice123** [comment]
I get what you mean about gl_outline—it can really make some models look off, like your grappling hook example. Having the option to exclude specific models would be ideal, so the outline only applies where it https://jackpotjill-casino-aus.com/ enhances the look rather than breaking it. That way, you could keep the effect for characters or objects that benefit from it, but leave tricky items untouched. It would give much more control and keep everything looking intentional instead of awkward.
